task3/index.html

```diff
@@ -0,0 +1,254 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Gektor</title>
+    <link rel="stylesheet" href="styles.css">
+
+</head>
+        <body>
+            <div class="leftcol">
+                <header>
+                    <img class="logo" src="logo.jpg" align="center" alt="logotip"> <strong>Gektor</strong>
+                    <p class="username"><strong>Anastasiya Zyatkova</strong></p>
+                </header>
+                <div id="menu">
+                    <p><img class="picbut" src="home.png" alt="home" align="center"><button class="menubut"><strong>Home</strong></button></p>
+                    <p><img class="picbut" src="profile.png" alt="profile" align="center"><button class="menubut"><strong>Profile</strong></button></p>
+                    <p><img class="picbut" src="logout.png" alt="log out" align="center"><button class="menubut"><strong>Log Out</strong></button></p>
+                </div>
+                <button class="note1"><strong>Note</strong></button>
+            </div>
+
+            <div class="rightcol">
+                <div class="searchtags">
+                    <form>
+                        <input class="edit" id="searchtags" type="search"  placeholder="Search by tags..">
+                    </form>
+                </div>
+                <div class="filtred">
+                    <form>
+                        <p class="filt"><strong>Filtred by...</strong></p>
+                        <p><input class="edit" id="author" type="search" placeholder="Author"></p>
+                        <p><input class="edit" id="date" type="date" placeholder="Date"></p>
+                        <p><input class="edit" id="tag" type="search" placeholder="Tag"></p>
+                    </form>
+                </div>
+                <footer>
+                    <p>Anastasiya Zyatkova</p>
+                    <p>2 course 7 group</p>
+                    <p>na.zzzya17@gmail.com</p>
+                    <p>© Gektor, Inc., 2020</p>
+                </footer>
+            </div>
+
+            <div class="middlecol">
+                <div>
+                    <p  class="namepage"><strong>Home</strong></p>
+                </div>
+                <div class="postnew" align="top">
```

Comment:
Don't use align it's for "Rich(formatted) Text features" we should provide layout trough plain css